editor/src/clj/editor/code/script_compilation.clj

```diff
@@ -68,12 +68,13 @@
 (def resource-kind->workspace->extensions
   "Declares which file extensions are valid for different kinds of resource
   properties. This affects the Property Editor, but is also used for validation."
-  {"atlas"       #(workspace/resource-kind-extensions % :atlas)
-   "font"        (constantly "font")
-   "material"    (constantly "material")
-   "buffer"      (constantly "buffer")
-   "texture"     (constantly (conj image/exts "cubemap" "render_target"))
-   "tile_source" (constantly "tilesource")})
+  {"atlas"        #(workspace/resource-kind-extensions % :atlas)
+   "font"          (constantly "font")
+   "material"      (constantly "material")
+   "buffer"        (constantly "buffer")
+   "texture"       (constantly (conj image/exts "cubemap" "render_target"))
+   "tile_source"   (constantly "tilesource")
+   "render_target" (constantly "render_target")})
```

Comment:
it's already available in the "texture", but since BOB allows this functionality out of the gate, the editor should to. Also, it serves different use cases